src/Annotation/ArrayValue.php

```diff
@@ -26,4 +26,10 @@ final class ArrayValue implements ValueInterface
      * @var bool
      */
     public ?bool $uniqueItems = null;
+
+    /**
+     * @codingStandardsIgnoreLine
+     * @var \ScrumWorks\OpenApiSchema\Annotation\ValueInterface we can't FQN, because shitty doctrine annotations
+     */
+    public ?ValueInterface $itemsSchema = null;
```

Comment:
A proc je tam vubec ta var annotace potreba? Kdyz cely phpdoc smazu, tak testy dal bezi

Reply:
protoze na zaklade toho doctrine anotace kontroluji vstup, jenze blbeci neumi pouzit anotaci z PHP, takze jim to musis dat tahkel (pokud chceme aby si to primo kontrolovali anotace) - mimojine o tom je zminka v doccomentu nahore nad entitou ze to je duvod proc tam ty @var anotace mame